config/locales/es.yml

```diff
@@ -2459,6 +2459,16 @@ es:
           title1: Tu declaración de impuestos federales ha sido transferida. Antes de que termines, tenemos algunas preguntas finales que pueden ayudar a maximizar tu reembolso.
           title2: Primero, necesitamos más información sobre ti y los miembros de tu familia.
           your_dob: Tu fecha de nacimiento
+      nc_eligibility_out_of_state_income:
+        edit:
+          out_of_state_income_html: "¿Usted (o su cónyuge, si presentan la declaración juntos) obtuvo algún ingreso en otro estado en %{filing_year}? (Esto incluye los ingresos por desempleo <strong>recibidos</strong> de otro estado)."
+          title: Primero, veamos si puedes usar esta herramienta para presentar tus impuestos.
+          withdraw_529_html: ¿Retiró usted (o su cónyuge, si presentan una declaración juntos) dinero de un <a href="https://www.cfnc.org/save-for-college/" target="_blank" rel="noopener nofollow">Cuenta de ahorros para la universidad 529</a> y usarla para gastos no calificados en %{filing_year}?
+      nc_eligibility_residence:
+        edit:
+          lived_in_state_html: "¿Usted (y su cónyuge, si presentan la declaración juntos) vivieron en %{state} durante <strong>todo</strong> el %{filing_year}?"
+          subtitle: Para la temporada de impuestos %{filing_year}, esta herramienta solo podrá atender a personas con situaciones tributarias específicas. Responda las siguientes preguntas para saber si puede utilizarlo.
+          title: Primero, veamos si puedes usar esta herramienta para presentar tus impuestos.
```

Comment:
will need a primer on how we translate things here

Reply:
@arinchoi03 we do integrate with transifex! we just put placeholder strings (google translated strings) for now and then eventually someone will translate in transifex and we will pull it into our app https://www.notion.so/cfa/Transifex-manual-sync-e5e04ebb6e1e49fa9bac8288c67cbae9